src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -493,6 +349,95 @@ bool Validator_30::ShouldUseOnlyConv2DGnaIface() const {
     return false;
 }
 
+class Validator_35 : public AbstractValidator {
+    struct CnnLimits {
+        const RangeLimit2D kInputHWLimit;
+        const RangeLimit kInputChannelsNumberLimit1B;
+        const RangeLimit kInputChannelsNumberLimit2B;
+        const RangeLimit kKernelNumberLimit;
+        const RangeLimit2D kKerneHWlLimit1B;
+        const RangeLimit2D kKerneHWlLimit2B;
+        const RangeLimit2D kStrideHWLimit1B;
+        const RangeLimit2D kStrideHWLimit2B;
+        const RangeLimit2D kDilationLimit;
+        const RangeLimit2D kPoolingWindowHWLimit;
+        const RangeLimit2D kPoolingStrideHWLimit;
+    };
+
+    static const CnnLimits kCnn2DLimits;
```

Comment:
I would put privarte fields and methods at the end of declaration order

Reply:
The same as above, I applied our coding standard rules only for Limitations class